Swiftfin tvOS/Views/ServerDetailView.swift

```diff
@@ -35,36 +35,49 @@ struct EditServerView: View {
                     .frame(maxWidth: 400)
             }
             .contentView {
-                Section(L10n.server) {
-                    TextPairView(
-                        leading: L10n.name,
-                        trailing: viewModel.server.name
-                    )
+
+                Section {
```

Comment:
Please use the other init instead. I know there are many places throughout that use the other one but I've been moving to this init instead.

```swift
Section(L10n.server) {
    // ...
}
```